app/views/articles/index.html.erb

```diff
@@ -1,6 +1,19 @@
-<ul>
-  <% @articles.each do |article| %>
-    <li><%= link_to article.title, article_path(article) %></li><br />
+
+<div class="jumbotron">
```

Comment:
Please remove empty lines and indent this file properly

Reply:
ok